client/src/app/app.module.ts

```diff
@@ -91,13 +95,15 @@ import { RecitationsComponent } from './modules/pages/recitations/recitations.co
     MatListModule,
     MatButtonModule,
     ModalModule.forRoot(),
-    MatTabsModule
+    MatTabsModule,
+    NgxDatatableModule,
   ],
   providers: [
     UserService,
     CourseService,
     SectionService,
     MeetingTimeService,
+    { provide: HTTP_INTERCEPTORS, useClass: TokenInterceptor, multi: true },
```

Comment:
whats this for?

Reply:
Handles adding tokens to HTTP requests for you